src/config/default-app-config.ts

```diff
@@ -516,6 +516,7 @@ export class DefaultAppConfig implements AppConfig {
       enabled: false,
       filter: ['title', 'author', 'subject', 'entityType'],
     },
+    defaultFilterCount: 5,
```

Comment:
We should add an example of this new setting also to the `config.example.yml` with an explanation of what it does and why you might change it.